tls-client/main.cpp

```diff
@@ -375,6 +387,7 @@ class HelloHTTPS {
         if(NSAPI_ERROR_WOULD_BLOCK == recv){
             return MBEDTLS_ERR_SSL_WANT_READ;
         }else if(recv < 0){
+            mbedtls_printf("Socket recv error %d\r\n", recv);
             return -1;
```

Comment:
Did not intend to change this. This could be pat of a different PR after analysing why it was done like this originally.